cub/cub/device/dispatch/dispatch_segmented_sort.cuh

```diff
@@ -200,13 +201,14 @@ CUB_RUNTIME_FUNCTION _CCCL_VISIBILITY_HIDDEN cudaError_t DeviceSegmentedSortCont
  * Continuation kernel is used only in the CDP mode. It's used to
  * launch DeviceSegmentedSortContinuation as a separate kernel.
  */
-template <typename ChainedPolicyT,
+template <typename WrappedPolicyT,
```

Comment:
Question: Why do we need to pass a wrapped policy to a kernel? This is new. So far kernels always retained their ChainedPolicy parameter.

Reply:
This is a special kernel used only to launch the segmented sort kernels when CDP is enabled. It is meant to be called with one thread only.